package.json

```diff
@@ -137,6 +137,7 @@
     "vscode:prepublish": "cd client && npm run update-vscode && cd .. && npm run compile",
     "compile": "tsc -b",
     "watch": "tsc -b -w",
-    "postinstall": "cd client && npm install && cd .."
+    "postinstall": "cd client && npm install && cd ..",
```

Comment:
I didn't noticed this before, you could remove your use of `cd` by using npm's `--prefix` flag and this script would look something like `npm install --prefix client`